javarosa/src/main/java/org/javarosa/core/util/SizeBoundVector.java

```diff
@@ -34,26 +30,14 @@ public int getAdditional() {
     }
 
     public void addBadImageReference() {
-        badImageReferenceCount++;
+        // TODO: remove this
```

Comment:
Got it, why are you removing the method body then? I don't understand the purpose of leaving the method in, but leaving it empty.